### PR TITLE
Chicken model and middlewares

### PR DESCRIPTION
**Pull request by AxelAlba**
- Added corresponding routes(CRUD) and model for the Chickens page.
- Added 2 middlewares: **''Cors"** to allow ajax requests from frontend to backend & **"morgan"** for automatic http requests logger

### CODE REVIEW

File: client/vue.config.js [hunk 1]
```diff
@@ -2,12 +2,14 @@ module.exports = {
...
-        changeOrigin: true
-      },
+  configureWebpack: {
+    devServer: {
+      proxy: {
+        '/api': {
+          target: 'http://localhost:3000',
+          changeOrigin: true
+        },
+      }
```
Comment (danjohnvelasco) [inline client/vue.config.js:12]: Ano difference nito? We're not pretty sure how this proxy thing works. Na try mo na ba?

**Review by danjohnvelasco — APPROVED** [review]
Looks good. I added a comment on vue.config.js but it's not crucial enough to delay merge.

File: client/vue.config.js (hunk 1 above)
Comment (AxelAlba) [reply client/vue.config.js:12]: Para di na mag add ng localhost:3000 to every request may proxy tayo na '/api' which denotes na if nag /api ka, deretso na sa http://localhost:3000/api yung request. Gagana lang to after mag configure ng webpack which is always the case if irurun mo na yung project.

Comment (danjohnvelasco) [reply client/vue.config.js:12]: i see. 

so /api/users --> localhost:3000/users, right?

**Pull request merged**